examples/chapter1/CMakeLists.txt

```diff
@@ -110,7 +132,7 @@ TARGET_LINK_LIBRARIES(${PROJ_NAME}_5b
 
 TARGET_INCLUDE_DIRECTORIES(${PROJ_NAME}_5b
   PRIVATE
-  ${CMAKE_CURRENT_SOURCE_DIR}/include 
```

Comment:
Did you do this format fixes manually or with the make format tool? Because we use clang-format for the code and do not allow manual format fixes.

Reply:
vscode removed a trailing white space in multiple lines. I did not even notice: vscode diff does not even show it.
No issues with the code: vscode follows .clang-format and I do run an automatic format before committing.